spec/features/system/admin/course_management_spec.rb

```diff
@@ -27,6 +32,14 @@
         end
       end
 
+      scenario 'I can only view active courses' do
```

Comment:
All courses in the system are displayed here. If we want to check, need to loop through all the inactive courses, that's costly. 

Actually, most of the time when we see the `active_course` in the first page, that means the feature is working.  otherwise, it will go to the last page.